### PR TITLE
Private Events, Ruby on Rails

### PR DESCRIPTION
**Pull request by Abdoulaye-Thespy**
In this milestone:

-We created a user sign up page
-We did a simple login for using the ID, which will be shown to you when you sign up.
- Users are able to create events and register for an event that they would like to attend.
- List of attendees are listed under the event itself on the show page.

### CODE REVIEW

**Comment by uryelah** [comment]
Hi **Abdoulaye Razak**, I'm Sarah your reviewer for this PR 👩‍🚀 

You did a great job with this project, good attention to detail! There are a couple of issues to fix before going forward but considering the work you already put in the project, it will be a breeze for you! 💪 

# Required Changes 🏗️ 🐛 💡

- [ ] Make sure that any ActiveRecord or SQL query, like .where .select .order, etc, is in model methods and not directly in the controllers.

- [ ] Refactor the view logic, conditionals and loops, out of the views and into helper methods.


-----

 **Kindly fix the pointed out issues before requesting the next code review**

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: app/models/event.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    Event.where('date >= ?', Date.today)
+  end
+
+  def self.past
+    Event.where('date < ?', Date.today)
+  end
+
+  scope :coming, -> { where('date >= ?', Date.today) }
+  scope :past, -> { where('date < ?', Date.today) }
+end
```
Comment (mosesogwo) [inline app/models/event.rb]: Do we need both the scopes and the methods?

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello! :clap: We still have logic in the views that could be factored out into helpers:**
See your events index view and event show view.

**Optional:** You could also clean up indentation in some of your views

**Please take care of it before submitting another code review request.**

### Status: Changes requested

**Review by mosesogwo — APPROVED** [review]
**Congratulations! Your project is approved.**
Please take time to find out how you can neatly take out logic from your views. You can extract large blocks in a conditional to a partial and use a helper method to render if the condition is met.

_**Good luck in your next project**_

### Status: Approved :heavy_check_mark:

**Pull request merged**